shell/packages/sandstorm-db/db.js

```diff
@@ -966,7 +966,7 @@ SandstormDb = function (quotaManager) {
     settings: Settings,
     migrations: Migrations,
     // Omitted: StaticAssets
-    // Omitted: AssetUploadTokens
+    assetUploadTokens: AssetUploadTokens,
```

Comment:
How about I make a `Symbol` and store `AssetUploadTokens` on the SandstormDb instance at that symbol?

Perhaps I should do that for the other hidden collections as well?  Do you prefer a single hidden object behind one Symbol, or several hidden objects, each behind its own Symbol?  (I should do the same for `StaticAssets` too, I imagine.)

That said, I have found it useful in the past to be able to navigate to collection objects when debugging (especially since I haven't yet found a way to extract them from Meteor's internals).  For example, when I was attempting to replicate Oasis's plans for a local blackrock development instance, I wanted to access the `Plans` collection from the console, which I could only get a handle to through `globalDb.collections.plans`.